bot/commands.py

```
@@ -1,300 +1,471 @@
+            )
+            return None
+
+    def validate_fixed_update_time(self, value):
+        if value.upper() == "XX:XX":
             return None
+        try:
+            return datetime.strptime(value, "%H:%M").time()
+        except ValueError:
+            raise ValueError(
+                f"Invalid time format: {value}. Use HH:MM or XX:XX to disable."
+            )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)